### ISSUE TITLE
New terminology

### ISSUE THREAD

**Issue by chris-belcher** [AdamISZ/CoinSwapCS#24]
I think our current terminology of TX0/TX1/TX2/TX3/TX4/TX5 is not descriptive enough. The transactions just given numbers as labels with no clue as to their purpose.

Coinswap is quite hard to follow and it would be easier if there were more descriptive names.

Taking some inspiration from the [lightning network spec document](https://github.com/lightningnetwork/lightning-rfc/blob/master/00-introduction.md#glossary-and-terminology-guide). I propose these new names:

TX0 = Alice's funding transaction
TX1 = Carol's funding transaction
TX2 = Alice's backout transaction OR Alice's refund transaction
TX3 = Carol's backout transaction OR Carol's refund transaction
TX4 = Alice's settlement transaction (The coins go FROM Alice's multisig address TO Carol)
TX5 = Carol's settlement transaction

Of course I'm not wedded to any of these and am happy to use other names we come up with.

Also, I propose one of the entities is renamed to a male name. (Adam/Carol, Alex/Carol, Alice/Charlie or Alice/Chris). Then we can use gendered pronouns to make it a lot easier to talk about them. e.g. *"If he does this thing then she will do that thing"*.

**Comment by AdamISZ** [comment]
Thanks for the suggestions, I can certainly see the argument :)

**Comment by AdamISZ** [comment]
Closing because I'm not planning to do anything about it, rightly or wrongly, for now.
